tf/tfprocess.py

```diff
@@ -741,8 +858,12 @@ def calculate_test_summaries_v2(self, test_batches, steps):
         sum_policy_ul = 0
         for _ in range(0, test_batches):
             x, y, z, q, m = next(self.test_iter)
-            policy_loss, value_loss, moves_left_loss, mse_loss, policy_accuracy, value_accuracy, moves_left_mean_error, policy_entropy, policy_ul = self.calculate_test_summaries_inner_loop(
-                x, y, z, q, m)
+            if self.strategy is not None:
+                policy_loss, value_loss, moves_left_loss, mse_loss, policy_accuracy, value_accuracy, moves_left_mean_error, policy_entropy, policy_ul = self.strategy_calculate_test_summaries_inner_loop(
+                    x, y, z, q, m)
+            else:
+                policy_loss, value_loss, moves_left_loss, mse_loss, policy_accuracy, value_accuracy, moves_left_mean_error, policy_entropy, policy_ul = self.calculate_test_summaries_inner_loop(
+                    x, y, z, q, m)
             sum_policy_accuracy += policy_accuracy
```

Comment:
This block of delta - needs to be replicated added under the calculate validation code further down.  Validations were added since my original PR - and I guess you don't have them configured.